ietf/secr/sreq/tests.py

```diff
@@ -146,7 +146,7 @@ def test_edit(self):
         self.assertRedirects(r, redirect_url)
 
         # Check whether updates were stored in the database
-        sessions = Session.objects.filter(meeting=meeting, group=mars)
+        sessions = Session.objects.filter(meeting=meeting, group=mars).order_by("id")  # order to match edit() view
```

Comment:
In case anyone looks back and notices this PR does more than remove the pin - this test happened to show instability while testing removal of the pin. We accepted taking the fix as part of this PR intentionally.